website/blog/2022-07-27-getting-started-with-the-dbt-semantic-layer.md

```diff
@@ -0,0 +1,165 @@
+---
+title: "Getting Started With The dbt Semantic Layer"
+description: "There's been a lot of discussion around dbt Metrics and what they are. Callum Mccann clarifies what the Semantic Layer is, what is available today, and how one can already begin to use parts of it!"
+slug: getting-started-with-the-dbt-semantic-layer
+
+authors: [callum_mccann]
+
+tags: [dbt product updates]
+hide_table_of_contents: false
+
+date: 2022-07-27
+is_featured: true
+---
+
+# Getting Started With The dbt Semantic Layer
+
```

Comment:
Did we want to do the callout that this was for advanced users and that a more general guide to using metrics will be available in the coming months?

Reply:
I'm hesitant to do so because that is very much on the to-do list and needs a LOT of work